src/mscorlib/src/System/Collections/Comparer.cs

```diff
@@ -41,18 +41,6 @@ public sealed class Comparer : IComparer , ISerializable
             Contract.EndContractBlock();
             m_compareInfo = culture.CompareInfo;
         }
-        
-        private Comparer(SerializationInfo info, StreamingContext context) {            
```

Comment:
It's used in several places in Array and in some other types so I believe I have to keep it.

I brought back this with the other serialiation stuff.